src/main/resources/resources/logisim/strings/gui/gui_de.properties

```diff
@@ -747,7 +746,7 @@ startupQuitButton = Beenden
 # start/TtyInterface.java
 #
 loadIoError = Fehler beim Lesen der Datei mit Speicherabbild
-loadNoRamError = Kein RAM für die "-load"-Option gefunden.
+loadNoRamError = Kein MEM für die "-load"-Option gefunden.
```

Comment:
Is `MEM` widely used word for memory? If not, please use full word.

Reply:
Thanks for the feedback, will spell it out as Memory everywhere.

Reply:
I think you missed the one in strings/gui/gui.properties.

Reply:
Sorry about missing this one. I've just fixed it.